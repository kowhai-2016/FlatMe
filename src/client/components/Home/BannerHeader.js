import React from 'react'

const BannerHeader = () => {
  return (
    <div className='header'>
      <div className='container description'>
        <h1>FlatMe</h1>
        <h3>resolves all your flatting issues</h3>
        <hr />
        <h2>Make the Rules, Share the load, Live the dream!</h2>
        <h3>Creates transparency, accountability and harmoney between flatmates to maintain ideal living conditions</h3>
      </div>
    </div>
  )
}

export default BannerHeader
