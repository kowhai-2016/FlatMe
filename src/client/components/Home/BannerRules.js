import React from 'react'

const BannerRules = () => {
  return (
    <div className='container'>
      <h1>"Make the Rules"</h1>
      <p>contract generator at its finest</p>
    </div>
  )
}

export default BannerRules
