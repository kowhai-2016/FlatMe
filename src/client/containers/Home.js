import { connect } from 'react-redux'

import Home from '../components/Home'

const mapStateToProps = state => {
  return {
    account: state.account
  }
}

export default connect(mapStateToProps)(Home)
