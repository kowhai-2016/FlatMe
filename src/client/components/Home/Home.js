import React from 'react'

import BannerHeader from './BannerHeader'
import BannerRules from './BannerRules'

const Home = () => {
  return (
    <div>
      <BannerHeader />
      <BannerRules />
    </div>
  )
}

export default Home
