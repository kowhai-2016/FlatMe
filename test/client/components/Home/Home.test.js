/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import Home from '../../../../src/client/components/Home'

test('Home renders logout correctly', () => {
  const tree = renderer.create(
    <Home account={{}} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
