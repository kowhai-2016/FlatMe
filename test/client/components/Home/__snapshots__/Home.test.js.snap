exports[`test Home renders logout correctly 1`] = `
<div>
  <div
    className="Landing">
    <div
      className="header">
      <div
        className="description1">
        <h1
          id="mainTitle">
          goFlat
        </h1>
        <h3
          id="subtitle">
          resolves all your flatting issues
        </h3>
      </div>
      <br />
      <br />
      <div
        className="row description2">
        <div
          className="col-md-12">
          <h2>
            Make the Rules, Share the Load, Live the Dream!
          </h2>
          <h3>
            creates transparency, accountability and harmony between flatmates to maintain ideal living conditions
          </h3>
        </div>
      </div>
    </div>
    <div
      className="height">
      <div
        className="container">
        <div
          className="row">
          <div
            className="col-md-3">
            <img
              src="/images/img1.jpg"
              width="100%" />
          </div>
          <div
            className="col-md-9 leads">
            <h1>
              \"Make the Rules\"
            </h1>
            <p>
              leave notes for your flatmates, 
              <br />
               get the most of your sharing space
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      className="height share">
      <div
        className="container alternate">
        <div
          className="row">
          <div
            className="col-md-9">
            <h1>
              \"Share the Load\"
            </h1>
            <p>
              split your bills, schedule the chores 
              <br />
               and manage your documents
            </p>
          </div>
          <div
            className="col-md-3 leads">
            <img
              className="landing-img"
              src="/images/img2.jpg"
              width="100%" />
          </div>
        </div>
      </div>
    </div>
    <div
      className="height">
      <div
        className="container">
        <div
          className="row">
          <div
            className="col-md-3 leads">
            <img
              className="landing-img"
              src="/images/img3.jpg"
              width="100%" />
          </div>
          <div
            className="col-md-9 leads">
            <h1>
              \"Live the Dream\"
            </h1>
            <p>
              host a party, cook dinner, leaving for a long weekend, 
              <br />
               notify your flat
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
